src/common/Exceptionhandler/exceptionFIlter.ts

```diff
@@ -1,33 +1,66 @@
-import { ExceptionFilter, Catch, ArgumentsHost, HttpException, HttpStatus } from '@nestjs/common';
+import { ExceptionFilter, Catch, ArgumentsHost, HttpException, HttpStatus, UnauthorizedException} from '@nestjs/common';
 import { Response } from 'express';
+import { QueryFailedError, EntityNotFoundError, CannotCreateEntityIdMapError } from 'typeorm';
```

Comment:
@arati-tekdi we dont want to use typeorm. Please use something from nestjs common. You can check with @souravbhowmik1999